### ISSUE TITLE
Convert Figure 47 to Prefigure

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/library#979]
[Figure 47](https://tbil.org/preview/linear-algebra/MX1.html#MX1-adjacency) should be converted to Prefigure so it can be navigated by a keyboard, which is superior to just describing it with  text.  Also, if the layout could be rotated 90 degrees (so it is wider than it is tall) it will fit better in the print version.

**Comment by siwelwerd** [comment]
The graphs in [Section A.2: Computer Science: PageRank](https://tbil.org/preview/linear-algebra/pagerank.html#pagerank) 
should probably also be made into Prefigures for the same reason, though they don't have the same layout problems
